.github/workflows/conformance-gateway-api.yaml

```
@@ -81,8 +81,24 @@ jobs:
         with:
           sha: ${{ inputs.SHA || github.sha }}
 
+  wait-for-images:
+    name: Wait for images
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - name: Checkout context ref (trusted)
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+        with:
+          ref: ${{ inputs.context-ref || github.sha }}
+          persist-credentials: false
+      - name: Wait for images
+        uses: ./.github/actions/wait-for-images
+        with:
+          SHA: ${{ inputs.SHA }}
```

Shouldn't we add

```suggestion
          SHA: ${{ inputs.SHA }}
          images: cilium-ci operator-generic-ci
```

so the workflow is only waiting for the required images?

---

oh, nice catch. Yes